app/src/main/java/com/puzzlebench/clean_marvel_kotlin/data/service/response/CharacterAditionalInfoResponse.kt

```diff
@@ -0,0 +1,9 @@
+package com.puzzlebench.clean_marvel_kotlin.data.service.response
+
+class CharacterAditionalInfoResponse(
+        //val character: Character,
```

Comment:
is this necessary? if not, delete it pls

Reply:
done